cmd/soroban-cli/src/commands/contract/deploy/wasm.rs

```diff
@@ -100,19 +108,91 @@ pub enum Error {
     Network(#[from] network::Error),
     #[error(transparent)]
     Wasm(#[from] wasm::Error),
+    #[error("cannot access config dir for alias file")]
+    CannotAccessConfigDir,
+    #[error(
+        "alias must be 1-30 chars long, and have only letters, numbers, underscores and dashes"
+    )]
+    InvalidAliasFormat { alias: String },
+    #[error(transparent)]
+    JSONSerialization(#[from] serde_json::Error),
+    #[error(transparent)]
+    IO(#[from] std::io::Error),
```

Comment:
```suggestion
    JsonSerialization(#[from] serde_json::Error),
    #[error(transparent)]
    Io(#[from] std::io::Error),
```
Because in Rust caps are reserved for statics and consts, and single letter caps are reserved for generics.